docs/README.md

```diff
@@ -0,0 +1,20 @@
+# Docs
+
+`docs/` contains end user documentation geared towards writing smart contracts with PyTeal.  The README explains how to edit docs.
+
+## Local testing
+
+Pyteal uses [Sphinx](https://github.com/sphinx-doc/sphinx) to generate HTML from RST files.  Test changes by generating HTML and viewing with a web browser.
+
+Here's the process:
+* Perform one-time environment setup:
+  * Create a virtual environment isolated to `docs/`.  Do _not_ mix with a top-level virtual environment.
+  * Install dependencies via `pip`.
+* Generate HTML docs:  `make html`.
+* If successful, generated HTML is available in `docs/_build/html`.
+
+Additionally, each PR commit generates HTML docs via [Github Actions](../.github/workflows/build.yml).
+
+## Releasing
+
+Production docs live at https://pyteal.readthedocs.io/en/stable/.  [Github Actions](../.github/workflows/build.yml) automate the releasing process.
```

Comment:
Production docs are not actually built by this repo. Readthedocs has its own build process

Reply:
@jasonpaulos Sorry, I misread the `build.yml`.  In your mind, is it worth elaborating on the _readthedocs_ release process?

If yes and with a pointer, I'm happy to update the PR.  If not, I can remove the section.

Reply:
I think "Production docs live at https://pyteal.readthedocs.io/en/stable/" is sufficient for this section

Reply:
@jasonpaulos Sounds good - I took in the feedback via https://github.com/algorand/pyteal/pull/205/commits/eb6da01edd50efcd296155783950c335505e618f.